hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -293,7 +294,7 @@ boolean initInvocationTarget() {
         }
 
         targetMember = nodeEngine.getClusterService().getMember(invTarget);
-        if (!isJoinOperation(op) && targetMember == null) {
+        if (!(isJoinOperation(op) || isWanReplicationOperation(op)) && targetMember == null) {
```

Comment:
@jerrinot , For better delivery, we're changing the wan event delivery mechanism (for Batch and no-delay implementations) and new one depends on operations. Since target wan members are not members of the source cluster, we needed to introduce this change to allow WAN operations run on target cluster members.